OJP/OJP_Common.xsd

```diff
@@ -197,7 +197,10 @@
 			<xs:documentation>Reference to a DATED VEHICLE JOURNEY (Reference to a journey on a specific operating day)</xs:documentation>
 		</xs:annotation>
 		<xs:sequence>
-			<xs:element ref="JourneyRef"/>
+			<xs:choice>
+				<xs:element ref="JourneyRef"/>
+				<xs:element name="TrainNumber" type="xs:normalizedString"/>
```

Comment:
I really don't get why this is an "OR". Why wouldn't there be a JourneyRef for a train?